src/lib.rs

```diff
@@ -2,6 +2,7 @@ use im::{HashSet, Vector};
 use sha2::{digest::DynDigest, Digest, Sha256};
 use std::fmt::{Display, Formatter, Result};
 use std::io::{BufReader, Error, ErrorKind, Read, Result as IOResult};
+use tokio::io::AsyncReadExt;
```

Comment:
Completely optional, this trait can also be imported as:

`use tokio::prelude::*`

[Reference](https://docs.rs/tokio/0.2.9/tokio/io/trait.AsyncReadExt.html)

Reply:
I would have loved to use `prelude` but it seems to be missing from the current Tokio crate... :-(

Reply:
Curses! In that case, let's leave this the way it is :)